Topics/lifecycle.md

```diff
@@ -0,0 +1,111 @@
+A data analysis tool is useful.
+
+At this stage of the project, a key factor is the collaboration between data scientists and business users who, ultimately, are those with the most extensive knowledge in the field and are therefore those who will pave the way for success. From our experience, this collaboration is facilitated by data visualization tools. 
+
+Data visualization tools such as [Tableau](https://www.tableau.com/it-it) generally have direct access to various types of structured and unstructured data sources, so that they can be applied to raw data, and are extremely effective in identifying trends, anomalies or aberrant data in the analyzed data with a much higher level of productivity than the classic tabular approach.  
+
+For simple views, the Python [MatPlotLib](https://matplotlib.org/) library does a great job. We will see in a later guide how to use it.
+
+As mentioned above, we must bear in mind that a Data Science project must always be geared towards achieving results focused on the activity and have a global vision in line with the business strategy.
+
+In this regard, I would like to point out the project concept [S.M.A.R.T.](https://www.getorganizedwizard.com/smart-goals/an-overview-of-smart-project-management/), which helps to frame the problem and encourages wise evaluations during this phase.
+
+Machine learning projects must have short-term objectives and be managed with an agile approach: the recirculation of questions, hypotheses and related data must be continuous, new discoveries must be used to guide and improve subsequent series of projects and results, even partial, must be shared with stakeholders (the business, colleagues, or simply those interested).
+
+### Select a performance measure
+
+Choosing the appropriate metrics to measure the performance of a model is critical to effectively solve a problem through a Machine Learning approach. 
+
+Once the type of problem to be solved has been identified, it will be clear whether what you want to predict is a number (e.g., 4 percent rate) or a class chosen from a set (e.g., give input images, classify them according to the type of subject they portray).
+
+The first type of problem, where the output of the system I want to build is a number, is called **regression**, while the second is called **classification**.
+
+In Machine Learning problems, we are interested in measuring the performance of our system based on "how much is wrong" with respect to the correct output.
+
+In the case of regression problems, this measure of the error corresponds to the distance between the predicted value and the correct value, while in the case of classification problems it corresponds to measuring a concept called entropy between the aforesaid classes. 
```

Comment:
>  corresponds to measuring a concept called entropy between the aforesaid classes. 

This statement may need simplification i.e. maybe give examples and write in a simpler form.